src/popover/popover.directive.ts

```diff
@@ -84,31 +96,213 @@ export class PopoverContainer {
 	 * Emits an event when the state of `isOpen` changes. Allows `isOpen` to be double bound
 	 */
 	@Output() isOpenChange = new EventEmitter<boolean>();
-
+	/**
+	 * Show caret at the alignment position
+	 */
 	@HostBinding("class.cds--popover--caret") @Input() caret = true;
+	/**
+	 * Enable drop shadow around the popover container
+	 */
 	@HostBinding("class.cds--popover--drop-shadow") @Input() dropShadow = true;
-	@HostBinding("class.cds--popover--high-contrast") @Input() highContrast = false;
-	@HostBinding("class.cds--popover--open") @Input() isOpen = false;
-
+	/**
+	 * Enable high contrast for popover container
+	 */
+	@HostBinding("class.cds--popover--high-contrast") @Input() highContrast = true;
```

Comment:
shouldn't this remain false by default?

Reply:
@Licen-it, You're right, that's a miss on my end. I'm going to double check all of the attributes and make sure a fix is released ASAP. So sorry about this, this should not have happened.